templates/candlepin.container.epp

```diff
@@ -0,0 +1,27 @@
+<%- |
+  String[1] $image,
+| -%>
+### File managed with puppet ###
+## Module: '<%= $module_name %>'
+[Unit]
+Description=Candlepin
+After=local-fs.target
+
+[Install]
+WantedBy=multi-user.target default.target
+
+[Container]
+Image=<%= $image %>
+PodmanArgs=--cgroups=enabled
+LogDriver=journald
+Network=host
+
+Volume=/etc/tomcat/server.xml:/etc/tomcat/server.xml:Z
+Volume=/etc/tomcat/login.config:/etc/tomcat/login.config:Z
+Volume=/etc/tomcat/cert-roles.properties:/etc/tomcat/cert-roles.properties:Z
+Volume=/etc/tomcat/cert-users.properties:/etc/tomcat/cert-users.properties:Z
+Volume=/etc/tomcat/conf.d/jaas.conf:/etc/tomcat/conf.d/jaas.conf:Z
+Volume=/etc/tomcat/tomcat.conf:/etc/tomcat/tomcat.conf:Z
+Volume=/etc/candlepin:/etc/candlepin:Z
+Volume=/var/log/candlepin:/var/log/candlepin:Z
+Volume=/var/log/tomcat:/var/log/tomcat:Z
```

Comment:
Can we configure candlepin (and Tomcat) to log everything to the journal and avoid these log directories?

Reply:
Fair question - generally I think yes. There are some things already logged to the journal but not everything.